package.json

```diff
@@ -41,33 +41,35 @@
     "node-sass": "3.4.2",
     "postcss-loader": "0.7.0",
     "react-onclickoutside": "4.5.0",
-    "react-textarea-autosize": "3.3.0",
     "react-transform-catch-errors": "1.0.2",
     "react-transform-hmr": "1.0.4",
     "redbox-react": "1.2.2",
     "sass-loader": "3.1.2",
     "simperium": "0.2.4",
     "style-loader": "0.12.4",
-    "webpack": "1.12.14",
+    "webpack": "1.13.2",
```

Comment:
see my comment in the first post of this PR, can definitely roll back before this is merged